tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ts

```diff
@@ -283,6 +283,18 @@ export class ScalarCardComponent<Downloader> {
               selectedStepData.VALUE_CHANGE =
                 closestEndPoint!.value - closestStartPoint.value;
               continue;
+            case ColumnHeaders.START_STEP:
+              if (!closestEndPoint) {
```

Comment:
closestEndPoint is not used in the calculation for START_STEP. Is it necessary to check it here?

The only reason I could imagine it being here is because its shorthand for "This is a range selection property only" but the responsibility for deciding that is actually scalar_card_container's when it is calculating columnHeaders$.

Reply:
I agree this should not be there.